packages/nodejs/src/clientV2.ts

```diff
@@ -0,0 +1,167 @@
+import { ChannelCredentials, Metadata } from '@grpc/grpc-js';
+import {
+    AccountAddress as AccountAddressLocal,
+    CredentialRegistrationId as CredentialRegistrationIdLocal,
+} from '@concordium/common-sdk';
+import {
+    AccountAddress,
+    BlockHashInput,
+    CryptographicParameters,
+    Empty,
+    NextAccountSequenceNumber,
+    AccountInfoRequest,
+    AccountIdentifierInput,
+    AccountInfo
+} from "./grpc/v2/concordium/types"
+import { QueriesClient } from "./grpc/v2/concordium/service.client"
+import {GrpcTransport} from "@protobuf-ts/grpc-transport";
+
+export type AccountIdentifierInputLocal = AccountAddressLocal | CredentialRegistrationIdLocal | bigint
+
+//////////////////////
+// Helper Functions //
+//////////////////////
```

Comment:
This type of sectioning indicates that this perhaps should just be another file. :thinking:

Reply:
Moved to `util.ts`